documentation/src/main/paradox/index.md

```diff
@@ -0,0 +1,119 @@
+# Akka HTTP
+
+Client and server backed by [Akka HTTP](https://doc.akka.io/docs/akka-http/current/).
+
+@@@ warning
+
+As explained in the section
+[Mixed versioning is not allowed](https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed)
+in the Akka documentation, you have to make sure that all the
+Akka modules of your application have the same version.
+
+For this reason, the interpreters `akka-http-server` and
+`akka-http-client` have marked their dependency to Akka as “provided”.
+
+As a consequence, to use these interpreters you will have to
+explicitly add a dependency on `akka-stream`:
+
+~~~ scala
+"com.typesafe.akka" %% "akka-stream" % "<akka-version>"
+~~~
+
+Where `<akka-version>` is binary compatible and higher or equal to @var[akka-version].
+
+@@@
+
+## Client
+
+@coordinates[akka-http-client]
+
+@scaladoc[API documentation](endpoints4s.akkahttp.client.index)
+
+### `endpoints4s.akkahttp.client.Endpoints`
+
+The `Endpoints` interpreter fixes the `Endpoint[A, B]` type to a function
+from `A` to `Future[B]`:
+
+@@snip [Endpoints.scala](/client/src/main/scala/endpoints4s/akkahttp/client/Endpoints.scala) { #endpoint-type }
+
+This means that, given the following endpoint definition:
+
+@@snip [EndpointsDocs.scala](/client/src/test/scala/endpoints4s/akkahttp/client/EndpointsDocs.scala) { #endpoint-definition }
+
+It can be invoked as follows:
+
+@@snip [EndpointsDocs.scala](/client/src/test/scala/endpoints4s/akkahttp/client/EndpointsDocs.scala) { #invocation }
+
+### `endpoints4s.akkahttp.client.ChunkedEntities`
+
+The `ChunkedEntities` interpreter fixes the `Chunks[A]` type to `akka.stream.scaladsl.Source[A, _]`:
+
+@@snip [ChunkedEntities.scala](/client/src/main/scala/endpoints4s/akkahttp/client/ChunkedEntities.scala) { #stream-type }
+
+This means that, given the following endpoint definition:
+
+@@snip [ChunkedEntitiesDocs.scala](/algebras/algebra/src/test/scala/endpoints4s/algebra/ChunkedEntitiesDocs.scala) { #streamed-endpoint }
```

Comment:
@julienrf Any tips on how to deal with these links to original repo?

Reply:
What I did in the `play` repo is to copy-paste the endpoint definition so that the example is fully self-contained in the repo.

https://github.com/endpoints4s/play/blob/v5.0.0/client/src/test/scala/endpoints4s/play/client/EndpointsDocs.scala#L9-L10

We could do the same here.